xbmc/addons/kodi-addon-dev-kit/src/api2/kodi-addon-sharedlibrary/InterProcess.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+/*
+ *      Copyright (C) 2016 Team KODI
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with KODI; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "../version.h"
+#include "kodi/api2/definitions.hpp"
+
+#include KITINCLUDE(ADDON_API_LEVEL, AddonLib.hpp)
+#include KITINCLUDE(ADDON_API_LEVEL, .internal/AddonLib_internal.hpp)
+
+#include <stdlib.h>
+#include <string>
+
+#ifdef TARGET_WINDOWS
+#include <windows.h>
+#else
+#ifndef __cdecl
+#define __cdecl
+#endif
+#ifndef __declspec
+#define __declspec(X)
+#endif
+#endif
+
+extern "C"
+{
+
+  using namespace API_NAMESPACE_NAME::KodiAPI;
+
+  namespace ADDON
+  {
+    typedef struct AddonCB
+    {
+      const char* libBasePath;  ///< Never, never change this!!!
+      void*       addonData;
+    } AddonCB;
+  }
+
+  class CKODIAddon_InterProcess
```

Comment:
Why is this called InterProcess? I don't see any inter process communication here